javav2/example_code/iotsitewise/src/main/java/com/example/iotsitewise/scenario/CloudFormationHelper.java

```diff
@@ -0,0 +1,134 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.example.iotsitewise.scenario;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.cloudformation.CloudFormationAsyncClient;
+import software.amazon.awssdk.services.cloudformation.CloudFormationClient;
+import software.amazon.awssdk.services.cloudformation.model.Capability;
+import software.amazon.awssdk.services.cloudformation.model.CloudFormationException;
+import software.amazon.awssdk.services.cloudformation.model.DescribeStacksRequest;
+import software.amazon.awssdk.services.cloudformation.model.DescribeStacksResponse;
+import software.amazon.awssdk.services.cloudformation.model.Output;
+import software.amazon.awssdk.services.cloudformation.model.Stack;
+import software.amazon.awssdk.services.cloudformation.waiters.CloudFormationAsyncWaiter;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+public class CloudFormationHelper {
+    private static final String CFN_TEMPLATE = "SitewiseRoles-template.yaml";
+    private static final Logger logger = LoggerFactory.getLogger(CloudFormationHelper.class);
+
+    private static CloudFormationAsyncClient getCloudFormationClient() {
+        CloudFormationAsyncClient cfClient = CloudFormationAsyncClient.builder()
+            .region(Region.US_EAST_1)
+            .build();
+
+        return cfClient;
+    }
+
```

Comment:
make the asych client a static member and use it. Right now you are creating a new instance everytime you call `getCloudFormationClient()`.

Reply:
Will follow same pattern that we use in Actions class for a given client.